src/controllers/eventEmitter/eventEmitter.ts

```diff
@@ -100,9 +108,7 @@ export default class EventEmitter {
           // Silence this error in prod to avoid displaying wired error messages.
           // The only benefit of displaying it is for devs to see when an action is dispatched twice.
           // TODO: If this happens on PROD, ideally we should get an error report somehow somewhere.
-          process.env.APP_ENV === 'production' && process.env.IS_TESTING !== 'true'
-            ? 'silent'
-            : 'minor',
+          errorLevel,
```

Comment:
You moved the logic up, but left the comment explaining the reasoning here 👀 , maybe move the comment too?

Reply:
Fixed in https://github.com/AmbireTech/ambire-common/pull/1005/commits/76b5c28f75141ba1245ee80765f65ee73ec7e757